analyses/snv-callers/compare_snv_callers.Rmd

```diff
@@ -0,0 +1,484 @@
+  )
+```
+
+Make a barplot illustrating the percent of the mutations for each caller that 
+that are each type of change. 
+
+```{r}
+perc_change_df %>%
+  ggplot2::ggplot(ggplot2::aes(x = change, y = percent)) +
+  ggplot2::theme_classic() +
+  ggplot2::geom_bar(
+    position = "dodge", stat = "identity",
+    ggplot2::aes(fill = caller)
+  ) +
+  ggplot2::theme(axis.text.x = ggplot2::element_text(angle = 45, hjust = 1)) +
+  ggplot2::xlab("") +
+  ggplot2::ylab("Percent of callers' mutations") +
+  colorblindr::scale_fill_OkabeIto()
+```
+
+```{r}
+# Save this plot
+ggplot2::ggsave(file.path(consensus_plots_dir, "perc_base_change_plot.png"))
+```
+
+## VAF distributions
+
+```{r}
+vaf_df %>%
+  ggplot2::ggplot(ggplot2::aes(x = caller, y = vaf, color = caller)) +
+  ggplot2::geom_violin() +
+  ggplot2::theme_classic() +
+  colorblindr::scale_color_OkabeIto()
+```
+
+```{r}
+# Save this plot
+ggplot2::ggsave(file.path(consensus_plots_dir, "vaf_violin_plot.png"))
+```
+
+## TMB distributions
+
+```{r}
+tmb_df %>%
+  ggplot2::ggplot(ggplot2::aes(x = caller, y = tmb, color = short_histology)) +
+  ggplot2::geom_boxplot() +
+  ggplot2::theme_classic()
+```
+
+```{r}
+# Save this plot
+ggplot2::ggsave(file.path(consensus_plots_dir, "tmb_boxplot_histology_plot.png"))
+```
+
+## VAF correlations
+
+```{r}
+vaf_long <- vaf_df %>%
+  dplyr::group_by(mutation_id, caller) %>%
+  dplyr::summarize(vaf = mean(vaf)) %>%
+  dplyr::distinct(mutation_id, caller, vaf, .keep_all = TRUE) %>%
+  tidyr::spread(caller, vaf) %>%
+  tibble::column_to_rownames("mutation_id")
+```
+
+Make the plot with ggpairs. 
+
+```{r}
+GGally::ggpairs(vaf_long, mapping = ggplot2::aes(alpha = 0.05)) +
+  ggplot2::theme_classic()
+``` 
+
+```{r}
+# Save this plot
+ggplot2::ggsave(file.path(consensus_plots_dir, "vaf_correlations_plot.png"))
+```
+
+## TMB correlations
+
+```{r}
+tmb_long <- tmb_df %>%
+  dplyr::ungroup() %>%
+  dplyr::select(-genome_size, -experimental_strategy) %>%
+  dplyr::distinct(Tumor_Sample_Barcode, caller, tmb) %>%
+  tidyr::spread(caller, tmb) %>% 
+  dplyr::select(-Tumor_Sample_Barcode)
+```
+
+Make the TMB correlation plot.
+
+```{r}
+GGally::ggpairs(tmb_long, mapping = ggplot2::aes(alpha = 0.05), cardinality_threshold = NULL) +
+  ggplot2::theme_classic()
+```
+
+```{r}
+# Save this plot
+ggplot2::ggsave(file.path(consensus_plots_dir, "tmb_correlations_plot.png"))
+```
+
+## Save consensus mutation calls
+
+Because of VarDict's oversensitivity, we will ignore VarDict's calls and keep mutations that
+are agreed upon by Lancet, Mutect2, and Strelka2. 
+Let's get the mutation ids for these. 
+
+```{r}
+consen_mutations <- callers_per_mutation %>%
+  dplyr::filter(caller_combo == "lancet-mutect2-strelka2") %>%
+  dplyr::select(-caller_combo) %>%
+  dplyr::pull(mutation_id)
+```
+
+Isolate the mutations to only these mutations, use the Strelka2 stats. 
```

Comment:
I know we said use strelka stats, but we already calculated median VAF? Should we add that on?